README.md

```diff
@@ -60,39 +52,35 @@ If you're just getting started with Grafana Cloud, you can
 [sign up for permanent free plan](https://grafana.com/products/cloud/).
 
 1. Click **Details** in the **Grafana** section on <https://grafana.com/profile/org>
-2. Copy **Instance ID** and **Zone** into the java command below
-3. On the left side, click on **Security** and then on **API Keys**
-4. Click on **Create API Key** (MetricsPublisher role) and copy the key into the java command below
-
-Enable the instrumentation agent using the `-javaagent` flag to the JVM.
+2. Copy **Instance ID** and **Zone** into the command below.
+3. On the left menu, click on **Security** and then on **API Keys**
+4. Click on **Create API Key** (MetricsPublisher role) and copy the key into the command below
+5. Choose a **Service Name** to identify your service.
+6. Optionally, you can make it easier to filter your data in Application Observability by adding the following
+   attributes:
+   - **deployment.environment**: Name of the deployment environment (`staging` or `production`)
+   - **service.namespace**: An optional namespace for `service.name`
+   - **service.version**: The application version, to see if a new version has introduced a bug
+   - **service.instance.id**: The unique instance, e.g. the pod name (a UUID is generated by default)
+
+Enable the instrumentation agent using the `-javaagent` flag to the JVM [^1].
```

Comment:
Also mention "...flat to the JVM with the following environment variables."

Reply:
Like so?

Add the `-javaagent` option to the java command with the following environment variables:

Reply:
That looks good, we're trying to explain that the agent looks for those env vars 👍